hack/create-bundle.sh

```
@@ -50,14 +59,39 @@ rm $MANIFEST.bak
+EOF
+
+if [ -n "$SKIP_RANGE" ]; then
+	printf '    skipRange: "%s"\n' "$SKIP_RANGE" >> $CATALOG
+fi
+
+$OPM_BIN render $BUNDLE_IMAGE --output yaml >> $CATALOG
+$OPM_BIN validate $CATALOG_DIR
+$OPM_BIN generate dockerfile $CATALOG_DIR
+
+# Build the catalog image and push it
+$TOOL_BIN build -t $INDEX_IMAGE -f $CATALOG_DIR.Dockerfile .
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)